srcpkgs/neovim-gtk/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'neovim-gtk'
+pkgname=neovim-gtk
+version=0.2.0
+revision=1
+build_style=cargo
+makedepends="atk-devel cairo-devel gdk-pixbuf-devel glib-devel gtk+3-devel pango-devel"
+short_desc="GTK UI for Neovim"
+maintainer="FollieHiyuki <folliekazetani@protonmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/daa84/neovim-gtk"
+distfiles="https://github.com/daa84/neovim-gtk/archive/v${version}.tar.gz"
+checksum=df4567bc687074e085b16199cd63edd0b19c12a3d66721f8c2c6021e709f3a21
+
+post_install() {
+	vinstall ${wrksrc}/desktop/org.daa.NeovimGtk.desktop 644 usr/share/applications
```

Comment:
Why not use https://github.com/daa84/neovim-gtk/blob/master/Makefile#L15 ?

Reply:
It uses `/usr/local` by defaut.
I will rewrite now, since I saw `--prefix=/usr` already there in `configure_args`.
My bad for not reading things through.